Makefile

```diff
@@ -410,7 +410,7 @@ ENVTEST_K8S_VERSION = 1.28.3
 
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 # k8s deps @ 1.28.0 https://github.com/kubernetes-sigs/controller-tools
-CONTROLLER_TOOLS_VERSION ?= v0.13.0
+CONTROLLER_TOOLS_VERSION ?= v0.14.0
```

Comment:
controller-tools v0.14.0 has k8s deps at 1.29.

Not sure what issues we might have there, if any.

Reply:
I've added a more detailed comment regarding `CONTROLLER_TOOLS_VERSION`.

> but worth being cautious about..
We'll see on staging if it works 😉

Reply:
Haha - sounds good, thanks. I'll put images from this PR on staging today for testing, then approve assuming all looks good :)

Reply:
Thanks!

I thought we just merge it and deploy the next version to staging 🤣 
But yeah, deploying from the branch sounds even better!

Reply:
Tested it on staging -- looks good!